docs/tutorial/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use an official Python runtime as a base image
+FROM python:2
+
+# Set the working directory to /app
+WORKDIR /package
```

Comment:
The comment mentions `/app` but the command says `/package`. The comment is pretty redundant anyway, might as well delete it.